remappers/kubernetesmetrics/k8smetrics_test.go

```diff
@@ -0,0 +1,327 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package kubernetesmetrics
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/elastic/opentelemetry-lib/remappers/common"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/tj/assert"
```

Comment:
This seems like the wrong import? Other packages are using stretchr.
If the intent is to have requires, there's the [require package](https://pkg.go.dev/github.com/stretchr/testify/require) that would avoid adding another dependency to the repository.

Reply:
@dmathieu nw we crossed messages here. It is fixed now with latest commit